### ISSUE TITLE
Exceptions are always marked as success invocations

### ISSUE THREAD

**Issue by tiagobrito** [brefphp/bref#642]
when throwing any exception on php, for example:
```php 
throw new Exception('Ups!');
```

It's always identified as a success invocation on AWS monitor :

<img width="406" alt="Screenshot 2020-05-04 at 15 30 25" src="https://user-images.githubusercontent.com/1071148/80965968-36552680-8e1c-11ea-8338-617128cf665c.png">

I'm using Symfony Framework

**Comment by mnapoli** [comment]
Hi, thanks for the report but we would need more information:

- Bref version
- which runtime/layer is used
- where is the exception thrown

**Comment by tiagobrito** [comment]
Bref version: 0.5.24
runtime/layer: (bref:layer.php-74-fpm)
Exception is thrown in a regular Symfony Controller

**Comment by mnapoli** [comment]
OK, that is the expected behavior: Symfony catches exceptions to return a response (display their custom 500 error page).

I'm not sure there is anything we can do here :/

**Comment by deleugpn** [comment]
I remember investigating this about 6 months ago and I shared this with my team. Laravel has the same behavior: every API call will always result in a response. Maybe the HTTP Code is 500 or 400, but Lambda doesn't interpret that, it's part of Load Balancer / API Gateway. So long the Lambda returns a valid response, the services treat it as a successful execution. Since we use Load Balancer, I remember saying that we should watch the ALB 5xx Cloud Watch metric instead of failed invocations.

**Comment by tiagobrito** [comment]
> OK, that is the expected behavior: Symfony catches exceptions to return a response (display their custom 500 error page).
> 
> I'm not sure there is anything we can do here :/

yes, Symfony catches the exception... but it will respect the http return codes.

**Comment by mnapoli** [comment]
As @deleugpn said, the HTTP status code is not taken into account by Lambda.

In JS or other languages, if you catch exceptions to return a specific HTTP response, no error is counted.

It all boils down to: does your code (including your framework) returns an exception or not. Here, Symfony will catch it so there is no exception reaching Lambda.

> I remember saying that we should watch the ALB 5xx Cloud Watch metric instead of failed invocations.

Yes this is a good indicator. You can watch logs as well.

In short, you are in the same situation as on any other hosting provider: watch HTTP response codes as well as logs.

**Comment by tiagobrito** [comment]
I notice this because I was exploring canary deployments with `serverless-plugin-canary-deployments` but it only checks for lambda invocation errors, which doesn't seem to happen.

**Comment by mnapoli** [comment]
Yes, I think you can add custom rules for canary deployments so you could check the logs or the HTTP response codes.
